eo-phi-normalizer/app/Main.hs

```diff
@@ -647,8 +647,19 @@ main = withUtf8 do
               logStrLn [fmtTrim|{linesCombined}|]
               logStrLn "\\end{phiquation*}"
             logStrLn "\n\\end{document}"
-        | latex ->
+        | latex -> do
+            logStrLn
+              [fmtTrim|
+                % {ruleSetTitle}
+
+                \\documentclass{{article}}
+                \\usepackage{{eolang}}
+                \\begin{{document}}
+              |]
```

Comment:
These lines together with the line that prints "\n\\end{document}" can be moved into a separate function parameterized by an IO action. The function can be reused in the `chain && latex` case.